cloud-service-providers/aws/eks/perf/gen-ai-perf.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: triton
+  labels:
+    app: triton
+spec:
+  containers:
+  - name: triton
+    image: nvcr.io/nvidia/tritonserver:24.04-py3-sdk
+    resources:
+      limits:
+        nvidia.com/gpu: 1
```

Comment:
If this is just hitting the running NIM, do we need to give the Triton pod a GPU resource?